packages/kbn-es/src/serverless_resources/security_roles.json

```diff
@@ -76,11 +85,18 @@
       {
         "feature": {
           "ml": ["read"],
-          "siem": ["read", "read_alerts"],
+          "siem": ["read", "read_alerts", "endpoint_list_read"],
```

Comment:
This siem privilege would mean the `t1_analyst` would be able to access the endpoint list page, which the [FTR test](https://github.com/elastic/kibana/blob/8315d8c3fb0b88d3d7196f3e3e4c56894257e3a4/x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_permissions.ts#L68) currently  expects otherwise. I guess we should update the test per the new privilege. 
cc @paul-tavares